mitodl/ocw-studio
# add CKEditor, example / test page

#### Pre-Flight checklist


#### What are the relevant tickets?

for #39 

#### What's this PR do?

This PR adds a basic, default build of CKEditor to our project. It has, right now, no markdown support yet. You can test out the editor at `/markdown-editor`.

#### How should this be manually tested?

Make sure you can WYSIWYG! All the stuff there should work as you'd expect it to. The example is currently set up to log output HTML to the console, so you can inspect that to see changes happening as you edit.

#### Screenshots (if appropriate)

![Screenshot from 2021-02-22 15-03-30](https://user-images.githubusercontent.com/6207644/108763357-46f79a00-751f-11eb-9e9a-9336354a74ff.png)
